src/cartservice/Program.cs

```diff
@@ -110,16 +112,38 @@ static void Main(string[] args)
               (ServerOptions options) =>
               {
                 var redis = NewRedisConnection(options.Redis);
+                var resourceBuilder = ResourceBuilder.CreateDefault()
+                                    .AddEnvironmentVariableDetector()
+                                    .AddService("cartservice");
+	
+		var deployment_environment = Environment.GetEnvironmentVariable("OTEL_ENVIRONMENT");
```

Comment:
Still not indented correctly :) but code itself is correct.

Reply:
ty, fixed (I think)